node/node.go

```diff
@@ -439,6 +439,25 @@ func (n *Node) initNetworking() error {
 		)
 	}
 
+	// We allow nodes to gossip unknown ACPs in case the current ACPs constant
```

Comment:
I wonder if this check/log should be done in `config/config`. Found it "odd" that we checked some conditions there and some here 🤷

Reply:
But no sure if there is some pattern we are following of what we verify here vs there.

Reply:
I think it's here because we have a ref to the logger here

Reply:
As dan noted - this is done here because the logger is initialized. We aren't really "verifying" here... because we do still allow the node to include acps that haven't been included in the client yet